templates/scanagroempresa/0/docker-compose.yml

```diff
@@ -410,10 +410,8 @@ services:
       GEOSERVER_ADMIN_PASSWORD: ${GEOSERVER_ADMIN_PASSWORD}
       # Kobo API
       KOBO_API_URL: ${KOBO_API_URL}
-      KOBO_API_CLIENTID: ${KOBO_API_CLIENTID}
-      KOBO_API_CLIENTSECRET: ${KOBO_API_CLIENTSECRET}
-      KOBO_API_USER: ${KOBO_API_USER}
-      KOBO_API_PASSWORD: ${KOBO_API_PASSWORD}
+      KPI_URL: ${KPI_URL}
+      KOBO_API_TOKEN: ${KOBO_API_TOKEN}
```

Comment:
In case the token have weird character, we should quote it with ""
Otherwise if it is alphanumeric, I think we are fine.